assignment/src/middlewares/auth.js

```diff
@@ -0,0 +1,16 @@
+const config = require('config');
+const jwt = require('jsonwebtoken');
+
+module.exports = function(req, res, next) {
+    const token = req.header('x-auth-token');
```

Comment:
we can put all of our code in try catch

Reply:
In my projects I'm using a package 'express-async-errors', it helps us to write all our express routes without using try-catch blocks for convinience. It actually wraps our code in a try-catch block and if we encounter any error it'll pass control to next middleware without crashing the app. And that next middleware will handle the error.

In this project all express route errors are being handled by 'middlware/exception'.

I am not sure if it is recomended or not. Please let me know.